src/Services/Masa.Scheduler.Services.Server/Application/Jobs/JobCommandHandler.cs

```diff
@@ -0,0 +1,42 @@
+﻿namespace MASA.Scheduler.Service.Application.Jobs;
+
+public class JobCommandHandler
+{
+    private readonly JobDomainService _domainService;
+
+    public JobCommandHandler(JobDomainService domainService)
+    {
+        _domainService = domainService;
+    }
+
+    [EventHandler(Order = 1)]
+    public async Task CreateHandleAsync(JobCreateCommand command)
+    {
+        await _domainService.CreateJobAsync();
+        //you work
+        await Task.CompletedTask;
+    }
+}
+
+public class JobStockHandler : CommandHandler<JobCreateCommand>
```

Comment:
Why need a JobStock?

Reply:
It is the template create. I will changed when develop Job module